pkg/chunk/table_manager.go

```diff
@@ -39,11 +40,36 @@ var (
 		Name:      "dynamo_table_capacity_units",
 		Help:      "Per-table DynamoDB capacity, measured in DynamoDB capacity units.",
 	}, []string{"op", "table"})
+
+	expectedMinIndexTableNumber = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "cortex",
+		Name:      "expected_min_index_table_number",
```

Comment:
We're currently prefixing new metrics with the service name. I would suggest to add `table_manager_` prefix to all these metrics.